### PR TITLE
Test saved networks

### PR DESCRIPTION
**Pull request by mruwnik**
I wanted to check both that larger networks have the correct values (in case of refactors etc.) and also to check how fast they work, to have a benchmark for speed ups. That being said, the code is quite slow now, so I added an option to skip slow tests by default.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,13 @@ There are shell specific scripts to run the service, e.g. in the case of bourne
     pip install -r requirements-dev.txt
     python -m pytest
 
+#### Slow tests
+
+Some tests take a while to run (like > 1 minute), so are disabled by default. To run them,
+add the `--runslow` option, e.g.
+
+    python -m pytest --runslow
```
Comment (mruwnik) [inline README.md:39]: should this also be used on github actions? It'd slow down PRs etc., but will check things properly. Maybe only on merge to master?

File: tests/integration/ekn/test_networks.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import pytest
+from ekn.helpers import get_votes
+
+
+logger = logging.getLogger(__file__)
+data_dir = Path(__file__).parent.parent / 'data'
+
+
+def generate_network(make_network, title, n_users=150, prob_vote=0.2, max_votes=100):
+    """Make a network with random votes.
```
Comment (mruwnik) [inline tests/integration/ekn/test_networks.py:17]: this is a helper function used to generate the network files. It's not used, but it makes adding new files a lot easier, so I left it here

File: tests/integration/ekn/test_networks.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    ]
+    logger.info('got scores')
+
+    with open(data_dir / f'network-{title}.json', 'w') as f:
+        json.dump({'weights': raw_weights, 'scores': [i for i in scores if i[2] > 0]}, f)
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize('network_type', ('sparse', 'average', 'dense'))
+def test_saved_networks(network_type, make_network):
```
Comment (mruwnik) [inline tests/integration/ekn/test_networks.py:50]: or would this be better as a standalone script? This is nice, as it makes use of all the fixtures, but I'm not sure how much it fits as a test

**Review by tayler6000 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tayler6000) [reply README.md:39]: In theory it'll take more than a few minutes to review a PR anyways, so I'm not worried about slowing them down much.  I do think we should only run the slow comprehensive tests on PRs to master.

**Pull request merged**